main_Luca_A.py

```diff
@@ -0,0 +1,67 @@
+"""

```

Comment:
Good use of the docstring at the start of the file, I really like how you integrated the API call into the script.